### PR TITLE
Added repo for stunnel and showoff for rspec testig

### PR DESCRIPTION
**Pull request by josephoaks**
TRAINTECH-730 #Resolved #time 5m

### CODE REVIEW

File: .fixtures.yml [hunk 1]
```diff
@@ -20,6 +20,8 @@ fixtures:
     pe_r10k: "git@github.com:puppetlabs/puppetlabs-pe_r10k.git"
     pe_staging: "git@github.com:puppetlabs/puppetlabs-pe_staging.git"
     puppet_enterprise: "git@github.com:puppetlabs/puppetlabs-puppet_enterprise.git"
+    stunnel: "git@github.com:puppetlabs/puppetlabs-stunnel.git"
```
Comment (binford2k) [inline .fixtures.yml]: These can be forge modules instead of git repos. See #L26

Comment (binford2k) [inline .fixtures.yml]: ping @josephoaks

Comment (josephoaks) [inline .fixtures.yml]: There have been minor fixes to the repo and no new release to the forge, so I see no reason not to use the git repo vs forge

Comment (binford2k) [inline .fixtures.yml]: That's sort of the point. The module from the Forge is what will be used in the classroom.

Comment (binford2k) [inline .fixtures.yml]: Ha, I guess I used git urls for the others. I'll make a ticket to review this and decide on the best strategy.

**Pull request merged**